### PR TITLE
#27, #28 Add Title & Description validation cases

### LINKED ISSUE #27: Add description as critical validation case
Description is required as of https://github.com/nfdi4plants/archigator/issues/5

### LINKED ISSUE #28: [BUG] Title as critical validation case is missing
Title is required as of https://github.com/nfdi4plants/archigator/issues/5 and not present though expected!
Implement ASAP!

### PR DESCRIPTION
**Pull request by omaus**
We want to have a validation case where the Investigation Description is tested for presence. Also, we were shocked noticed that the validation case for Investigation Title is missing! To do so, this PR

- adds a validation case for the presence of the Description in the Investigation file as well as information extraction mechanisms to achieve that
- adds a validation case for the presence of the Title in the Investigation file as well as information extraction mechanisms to achieve that
- updates the arc-validation-result.xml file accordingly (for unit tests)
- closes #27 & closes #28

### CODE REVIEW

File: src/ArcValidation/Configs/ArcConfig.fs [hunk 1]
```diff
@@ -1,53 +1,55 @@
...
+
+    member this.AssayFolders = InformationExtraction.Assay.getFolders this.PathConfig.AssaysPath
+
+    member this.AssayFilesAndIds = InformationExtraction.Assay.getFilesAndIds this.AssayFolders
+
+    member this.AssayAnnotationTables = InformationExtraction.Assay.getAnnotationTables this.AssayFilesAndIds
+
+    member this.AssayRawOrDerivedDataPaths = InformationExtraction.Assay.getRawOrDerivedDataPaths this.AssayAnnotationTables
+
+    // Composite
```
Comment (kMutagene) [inline src/ArcValidation/Configs/ArcConfig.fs:53]: what's going on with this diff? I have absolutely no idea what changed :D

File: src/ArcValidation/InformationExtraction.fs [hunk 2]
```diff
@@ -1,160 +1,179 @@
...
+                fun ap ->
+                    let atd = try FsWorkbook.fromXlsxFile ap with :? IOException -> new FsWorkbook()
+                    let atdWorksheets = 
+                        let wss = 
+                            FsWorkbook.getWorksheets atd
+                            |> List.filter (fun ws -> ws.Name <> "Assay")
+                        wss |> List.iter (fun ws -> ws.RescanRows())
+                        wss
+                    let atdPathCvP = CvParam(Terms.filepath, ParamValue.Value ap)
+                    atdWorksheets
```
Comment (kMutagene) [inline src/ArcValidation/InformationExtraction.fs:160]: same here

File: src/ArcValidation/Configs/ArcConfig.fs (hunk 1 above)
Comment (omaus) [reply src/ArcValidation/Configs/ArcConfig.fs:53]: What I changed was just add this: https://github.com/nfdi4plants/arc-validate/blob/e25f7178954944fa4788ec35dbba156970c05e8d/src/ArcValidation/Configs/ArcConfig.fs#L25

File: src/ArcValidation/InformationExtraction.fs (hunk 2 above)
Comment (omaus) [reply src/ArcValidation/InformationExtraction.fs:160]: Lines added: https://github.com/nfdi4plants/arc-validate/blob/e25f7178954944fa4788ec35dbba156970c05e8d/src/ArcValidation/InformationExtraction.fs#L44

**Review by kMutagene — APPROVED** [review]

**Pull request merged**